OpenEdXMobile/src/main/java/org/edx/mobile/view/adapters/CourseOutlineAdapter.java

```diff
@@ -287,10 +289,23 @@ private void updateUIForVideo(@NonNull final ViewHolder viewHolder, @NonNull fin
         viewHolder.rowType.setIcon(FontAwesomeIcons.fa_film);
         viewHolder.numOfVideoAndDownloadArea.setVisibility(View.VISIBLE);
         viewHolder.bulkDownload.setVisibility(View.VISIBLE);
-
         viewHolder.rowSubtitlePanel.setVisibility(View.VISIBLE);
-        viewHolder.rowSubtitle.setVisibility(View.VISIBLE);
-        viewHolder.rowSubtitle.setText(videoData.getDurationReadable());
+        if (videoData.getDuration() > 0l) {
+            viewHolder.rowSubtitle.setVisibility(View.VISIBLE);
+            viewHolder.rowSubtitle.setText(videoData.getDurationReadable());
+        }
+        if (videoData.getSize() > 0l) {
```

Comment:
AS is giving quite the right suggestion. I too mistook it for a `1`. Better to replace `l` with `L`.

![screen shot 2017-10-03 at 3 58 15 pm](https://user-images.githubusercontent.com/1710804/31122112-117043aa-a854-11e7-9db8-a02f4e958186.png)